src/storage/localResourceLoader.ts

```diff
@@ -54,26 +47,6 @@ export class LocalResourceLoader extends ResourceLoader {
     return env.kafkaClusters;
   }
 
-  /**
```

Comment:
Like was done for `DirectResourceLoader`, remove this redundant implementation in favor of implementing in base class `ResourceLoader`.